mock.go

```diff
@@ -279,8 +280,10 @@ func (t *MockTable) keyFromColumnValues(values map[string]interface{}, keyNames
 
 	for _, keyName := range keyNames {
 		value, ok := values[keyName]
-
-		if !ok {
+		// need to explcitly check for string type here, as we cannot
+		// write an empty string in the primary key
+		stringVal, isString := value.(string)
+		if !ok || (isString && stringVal == "") {
```

Comment:
This is a hack. As far as I understand, it is safe to write a primary key with zero values for other types like int, time etc., and writes only fail for empty strings.